vip_hci/hci_dataset.py

```diff
@@ -772,12 +775,13 @@ def inject_companions(self, flux, rad_dists, n_branches=1, theta=0,
         Returns
         -------
         yx : list of tuple(y,x)
-            [if full_output] Pixel coordinates of the injections in the first
-            frame (and first wavelength for 4D cubes).
+            [full_output] Pixel coordinates of the injections in the first frame
```

Comment:
We haven't discussed thoroughly how to use the squared brackets in the docstrings. It's more informative to put [full_output=True] instead of just [full_output]

Reply:
👍